### PR TITLE
Add possibility to perform manual read & write tests

### PR DESCRIPTION
**Pull request by bartoszwilk**

### CODE REVIEW

File: example/lib/device_details/view/manual_test_view.dart [hunk 1]
```diff
@@ -96,8 +112,36 @@ class ManualTestView extends StatelessWidget {
...
+            ],
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.symmetric(vertical: 2.0),
+          child: Row(
+            children: <Widget>[
+              ButtonView("Write via peripheral", action: _writeCharacteristicForPeripheral),
+              ButtonView("Read via peripheral", action: _readCharacteristicForPeripheral),
+              ButtonView("Monitor via peripheral", action: null),
```
Comment (pawelByszewski) [inline example/lib/device_details/view/manual_test_view.dart]: `action: null` is useless

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 2]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
     log("negotiated MTU $negotiatedMtu");
   }
 
+  Future<void> readCharacteristicForPeripheral() async {
+    log("Reading temperature");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Reading temperature config`

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 3]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
...
+    CharacteristicWithValue readValue = await peripheral.readCharacteristic(
+        SensorTagTemperatureUuids.temperatureService,
+        SensorTagTemperatureUuids.temperatureConfigCharacteristic);
+    log("Temperature config value: ${readValue.value}");
+  }
+
+  Future<void> readCharacteristicForService() async {
+    log("Reading temperature");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Reading temperature config`

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 4]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
...
+        services.firstWhere((service) =>
+            service.uuid ==
+            SensorTagTemperatureUuids.temperatureService.toLowerCase()));
+    CharacteristicWithValue readValue = await service.readCharacteristic(
+        SensorTagTemperatureUuids.temperatureConfigCharacteristic);
+    log("Temperature config value: ${readValue.value}");
+  }
+
+  Future<void> readCharacteristic() async {
+    log("Reading temperature");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Reading temperature config`

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 5]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
...
+            characteristic.uuid ==
+            SensorTagTemperatureUuids.temperatureConfigCharacteristic
+                .toLowerCase());
+
+    Uint8List readValue = await characteristic.read();
+    log("Temperature config value: $readValue");
+  }
+
+  Future<void> writeCharacteristicForPeripheral() async {
+    log("Turning on temperature update");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Turning on temperature update on peripheral`

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 6]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
...
+        SensorTagTemperatureUuids.temperatureService,
+        SensorTagTemperatureUuids.temperatureConfigCharacteristic,
+        Uint8List.fromList([valueToSave]),
+        false);
+
+    log("Written \"$valueToSave\" to temperature config");
+  }
+
+  Future<void> writeCharacteristicForService() async {
+    log("Write to characteristic via service");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Turning on temperature update on service`

File: example/lib/test_scenarios/peripheral_test_operations.dart [hunk 7]
```diff
@@ -67,6 +67,90 @@ class PeripheralTestOperations {
...
+    await service.writeCharacteristic(
+        SensorTagTemperatureUuids.temperatureConfigCharacteristic,
+        Uint8List.fromList([valueToSave]),
+        false);
+
+    log("Written \"$valueToSave\" to temperature config");
+  }
+
+  Future<void> writeCharacteristic() async {
+    log("Write to characteristic directly");
```
Comment (mikolak) [inline example/lib/test_scenarios/peripheral_test_operations.dart]: `Turning on temperature update on characteristic`

**Pull request merged**